app/web/features/communities/events/CreateEventPage.test.tsx

```diff
@@ -71,6 +71,8 @@ describe("Create event page", () => {
       expect(titleInput).toHaveValue("Test event");
     });
 
+    // @TODO(NA): Need to add start and end dates now but it's triple tested in EventTimeChanger and I can't seem to type in them from here.
+
```

Comment:
@darrenvong Cry for help lol. Now that EventTimeChanger doesn't have default dates, I can't submit CreateEventPage. Even though I see the `startDate` field in `screen.debug`, it seems to be marked read-only (I assume because it's nested 2 levels deep: CreateEventPage => EventForm => EventTimeChanger)?

I *think* the answer is to just test the interaction that's actually happening in this component and strip out the submit functionality but I'm not sure?

Does anything jump out at you as a proper way to handle this? I'm stuck with this one.

With `EditEventForm` I could just pass default dates but obviously with a new event it's blank.

Reply:
omg nevermind it was that damn mui-x-datepickers mock FML